bootstrap/lib/mix/tasks/nerves/new.ex

```diff
@@ -3,7 +3,11 @@ defmodule Mix.Tasks.Nerves.New do
   import Mix.Generator
 
   @nerves Path.expand("../../../..", __DIR__)
-  @version "0.2"
+
+  @nerves_vsn "0.4.0"
```

Comment:
I think it's worth specifying them as all `x.y.z` unless there's a reason not to, so that we cause the least amount of surprise and let people choose when they want to upgrade to the next feature release.